qemu/README.md

```diff
@@ -0,0 +1,164 @@
+# QEMU
+
+<p align="center">
+    <img src="logo/400x127.png" alt="QEMU">
+</p>
+
+QEMU (stands for Quick EMUlator) is an open source emulator that allows mimicking many different processors and loading many different operating systems.
+
+The Dockerfile and assets related to it are intended primarily, but **not** exclusively, for building the Docker container which simplifies testing 64-bit Raspberry Pi images built by [Pieman](https://github.com/tolstoyevsky/pieman), but not limited by this.
+
+<table>
+  <tr>
+    <td align="center" colspan="2"><b>QEMU</b></td>
+  </tr>
+  <tr>
+    <td>Version</td>
+    <td><a href="http://lists.nongnu.org/archive/html/qemu-devel/2018-04/msg04089.html">2.12.0</a></td>
+  </tr>
+  <tr>
+    <td>Release date</td>
+    <td>24 Apr 2018</td>
+  </tr>
+  <tr>
+    <td>Data volume</td>
+    <td><code>current directory:/tmp</code></td>
+  </tr>
+  <tr>
+    <td valign="top">Base images</td>
+    <td>
+        <code>cusdeb/alpine3.7:armhf</code> (for <code>armhf</code> port)<br>
+        <code>cusdeb/alpine3.7:amd64</code> (for <code>amd64</code> port)
+    </td>
+  </tr>
+</table>
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Getting started](#getting-started)
+- [Documentation](#documentation)
+  * [CLI for the container](#cli-for-the-container)
+  * [The container entrypoint script](#the-container-entrypoint-script)
+- [Known issues](#known-issues)
+
+## Prerequisites
+
+`docker-qemu.sh`, which is described below, requires bash 4 or higher, so macOS users should upgrade their bash if they haven't done it yet.
+
+## Getting started
+
+First, build 64-bit custom image using Pieman or download one of the existing images from other projects (for example, you may find the 64-bit Gentoo image on [this page](https://github.com/sakaki-/gentoo-on-rpi3-64bit#gentoo-on-rpi3-64bit)). Then, get the `docker-qemu.sh` script
+
+```
+$ curl -O https://raw.githubusercontent.com/tolstoyevsky/mmb/master/qemu/docker-qemu.sh
+$ chmod +x docker-qemu.sh
+```
+
+or execute all the commands below from the `qemu` directory. Finally run the image in the emulator (assume the filename of the target image is `ubuntu-bionic-arm64-for-rpi3.img`).
```

Comment:
I'd suggest 'assume the filename of the target image is ubuntu-bionic-arm64-for-rpi3.img and it's located in qemu directory' or something like this cause it's not obvious for me that I cannot use absolute path like /home/denis/Documents/git/pieman/build/bionic-arm64-for-rpi3.img